### DISCUSSION TITLE
Organize extension methods under the type being extended

### DISCUSSION THREAD

**Discussion by xavierjohn** [dotnet/docfx#9197] [Q&A]
I would like to organize my extension method documentations under the type that is being extended.

For example: If I added extension methods to HttpResponse class then I would like to see all my extensions under that type

- HttpResponse
     - HttpHeaderExtension
     - HttpAuthExtension
     - Etc

Is this possible?

**Comment by mihtjel** [comment]
I have the same wish, but currently, looking at the code, it's not obvious how this would be done. I think it breaks down in `ExtractMetadataWorker` where the assemblies are processed, but any classes that are not in the source code won't have any metadata created at this point to export as YAML files.

I'm not savvy enough with the externals of the Metadata processing to implement it at this time, but I may take a shot at it eventually.
